tests/analysis/test_segmentation.py

```
@@ -545,3 +550,120 @@ def test_alternate_value_col(self, base_df):
         assert result_df.loc[2, "segment_name"] == "Light"
         assert result_df.loc[4, "segment_name"] == "Medium"
         assert result_df.loc[5, "segment_name"] == "Light"
+
+
+class TestRFMSegmentation:
+    """Tests for the RFMSegmentation class."""
+
+    @pytest.fixture()
```

Ruff (0.8.2)

Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)